src/components/backdrop/backdrop.js

```diff
@@ -66,6 +71,13 @@ angular
         }
       });
 
+      function resize() {
```

Comment:
Yes, a lot of CSS - but for support which I don't think is needed anymore. 
Point 3 is interesting, but maybe for legacy reasons. Anyway, I won't bother you anymore :)